sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java

```diff
@@ -55,21 +66,74 @@ public static BiConsumer<ExtendedRecord, EventCoreRecord> interpretEventType(
         interpretVocabulary(er, DwcTerm.eventType, vocabularyService).ifPresent(ecr::setEventType);
   }
 
-  /** {@link DwcTerm#lifeStage} interpretation. */
+  /** {@link DwcTerm#typeStatus} interpretation. */
+  public static BiConsumer<ExtendedRecord, BasicRecord> interpretTypeStatus(
+      VocabularyService vocabularyService) {
+    return (er, br) ->
+        extractListValue(er, DwcTerm.typeStatus)
+            .forEach(
+                value ->
+                    interpretVocabulary(
+                            DwcTerm.typeStatus,
+                            value,
+                            vocabularyService,
+                            v -> {
+                              if (SUSPECTED_TYPE_STATUS_VALUES.stream()
```

Comment:
SUSPECTED_TYPE_STATUS_VALUES.contains(v.toLowerCase())

Reply:
Or v is some long text?

Reply:
yeah, v can be a long text(e.g.: possibly allotype)